File: conn.js
const sqlite3 = require('sqlite3').verbose();

// Connect to the database
let db = new sqlite3.Database('./3alemni.db', (err) => {
    if (err) {
        return console.error('❌ Connection error:', err.message);
    }
    console.log('✅ Connected to the 3alemni SQLite database.');
});

// Example: Fetch all centers
db.all(`SELECT * FROM centers`, [], (err, rows) => {
    if (err) {
        throw err;
    }
    console.log('📚 centers:');
    rows.forEach((row) => {
        console.log(row);
    });
});

// Always close the connection
db.close((err) => {
    if (err) {
        return console.error(err.message);
    }
    console.log('🔌 Database connection closed.');
});

File: conn.py
import sqlite3

# Connect to the SQLite database
conn = sqlite3.connect('3alemni.db')  # Make sure the path is correct

# Create a cursor object to interact with the database
cur = conn.cursor()

# Example: fetch all centers
cur.execute("SELECT * FROM centers")
rows = cur.fetchall()

# Print the results
for row in rows:
    print(row)

# Always close the connection when done
conn.close()
